kafka-ui-react-app/src/components/Connect/List/__tests__/List.spec.tsx

```diff
@@ -52,20 +54,147 @@ describe('Connectors List', () => {
 
     it('opens broker when row clicked', async () => {
       renderComponent();
-      await userEvent.click(
-        screen.getByRole('row', {
-          name: 'hdfs-source-connector first SOURCE FileStreamSource a b c RUNNING 2 of 2',
-        })
-      );
-      await waitFor(() =>
+
+      const link = await screen.findByRole('cell', {name: 'hdfs-source-connector'});
+
+      await userEvent.click(link);
+      await waitFor(() => {
```

Comment:
@Mgrdich Could you please take a look on this test? I'm struggling to fix it even though the link itself works when tested manually

Reply:
or @sasunprov maybe you'll have some time since you're assigned to the issue.

Reply:
@kamsharipova we'll take a look :)

Reply:
maybe it works when you run it on its own, because there are globals mock that are not being reset-ted  normally in the upper `describe` block.

please check that and you can use `beforeEach` `afterEach`